prijateli_tree/app/templates/round.html

```diff
@@ -0,0 +1,46 @@
+{% extends "base.html" %}
+{% block title %}
+  PrijateliTree - View Round
+{% endblock title %}
+{% block content %}
+  <div style="text-align: center;">
+    <h1 style="margin-bottom: 30px;">Round: {{ current_round }}</h1>
```

Comment:
Use the standards that go with the h1, etc. classes.

Reply:
Could you explain this a bit more?